### PR TITLE
Add support for AWS ECR repositories, images, and vulnerability scan results

### PR DESCRIPTION
**Pull request by ngowraj**
PR to add ECR scans to Cartography. 
Repo => Images => Image Scan Results

### CODE REVIEW

File: cartography/intel/aws/ecr.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
+import logging
+
+from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+
+def get_ecr_repositories(boto3_session, region):
```
Comment (marco-lancini) [inline cartography/intel/aws/ecr.py:8]: Would it be possible to add the `@aws_handle_regions` decorator please?

File: cartography/intel/aws/ecr.py [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    client = boto3_session.client('ecr', region_name=region)
+    paginator = client.get_paginator('describe_repositories')
+    ecr_repositories = []
+    for page in paginator.paginate():
+        ecr_repositories.extend(page['repositories'])
+    return ecr_repositories
+
+
+def get_ecr_repository_images(boto3_session, region, repository_name):
```
Comment (marco-lancini) [inline cartography/intel/aws/ecr.py:17]: Would it be possible to add the `@aws_handle_regions` decorator please?

File: cartography/intel/aws/ecr.py (hunk 1 above)
Comment (achantavy) [reply cartography/intel/aws/ecr.py:12]: Done

**Comment by kunaals** [comment]
Looks really good to me. The docs are 💯

**Review by kunaals — APPROVED** [review]

**Pull request merged**